wandb/sdk/launch/create_job.py

```diff
@@ -335,10 +335,19 @@ def _create_repo_metadata(
         entrypoint = rel_entrypoint
 
     # check if requirements.txt exists
-    if not os.path.exists(os.path.join(local_dir, "requirements.txt")):
-        repo_formd = path.replace(entrypoint, "")
+    # start at the location of the python file and recurse up to the git root
+    req_dir = local_dir
+    while (
+        not os.path.exists(os.path.join(req_dir, "requirements.txt"))
```

Comment:
Do you know what happens when we traverse symlinks? Is it possible to pop out of a subdue into a symlinked dir and have it break?

Reply:
Like when we are inside of a symlinked dir

Reply:
🤔 Not sure if this interacts with symlinks at all since this only traverses directories that were present in the git repo. I tried running `wandb job create ...` inside a symlinked directory and the behavior was the same for all three cases - requirements.txt in the same subdir as the python file; in a parent directory; and not present at all.